launch

```diff
@@ -60,10 +70,21 @@ if __name__ == "__main__":
         pass
 
     if args.debug:
+
+        if not args.debugger:
+            args.debugger = 'cgdb'
+
+        if not args.debugger in DEBUGGERS:
+            raise SystemExit('Unrecognized debugger %s selected.' % args.debugger)
+
+        dbgcmd = DEBUGGERS[args.debugger]
```

Comment:
Please use https://docs.python.org/2/library/stdtypes.html#string-formatting-operations instead of `replace`.